specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -492,6 +635,203 @@
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherYear"
+          },
+          {
+            "$ref": "#/parameters/OptionalWeatherBasinId"
+          },
+          {
+            "$ref": "#/parameters/OptionalWeatherGovernmentId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormSearchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/forecasts/{format}": {
+      "get": {
+        "description": "**Government-issued forecasts**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all Government-issued forecasts.",
+        "operationId": "Weather_StormForecast",
+        "x-ms-examples": {
+          "StormForecast": {
+            "$ref": "./examples/GetStormForecast.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherYear"
+          },
+          {
+            "$ref": "#/parameters/RequiredWeatherBasinId"
+          },
+          {
+            "$ref": "#/parameters/RequiredWeatherGovernmentId"
+          },
+          {
+            "$ref": "#/parameters/WeatherDataUnit"
+          },
+          {
+            "$ref": "#/parameters/WeatherDetails"
+          },
+          {
+            "$ref": "#/parameters/WeatherRadiiGeometry"
+          },
+          {
+            "name": "windowGeometry",
+            "description": "When true, window geometry data (geoJSON) is included in the response",
+            "in": "query",
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormForecastResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/locations/{format}": {
+      "get": {
+        "description": "**Locations for an individual government-issued storm**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet locations of an individual government-issued storm.",
```

Comment:
Change:

Get locations of an individual government-issued storm.

To:

Get locations of an individual government-issued tropical storms. Information about the tropical storms includes, location coordinates, geometry, basin ID, date, wind details and wind radii.

Reply:
Updated